corehq/form_processor/submission_post.py

```diff
@@ -445,6 +448,34 @@ def do_post_save_actions(case_db, xforms, case_stock_result):
             })
             raise PostSaveError
 
+    @staticmethod
+    def send_to_elasticsearch(instance):
+        if instance.metadata.deviceID != FORMPLAYER_DEVICE_ID:
```

Comment:
we've seen some bugs where forms don't have metadata so best to guard this with a try catch or checking that `instance.metadata` is not None (See https://github.com/dimagi/commcare-hq/pull/31680)